woocommerce/woocommerce-android
# Navigate to new shipping label flow

Closes: #12444

### Description
This PR adds the necessary changes to navigate to the purchase shipping label flow.
 
1. When the legacy WCS&T plugin is active, the app will navigate to the old flow
2. When the new WCS plugin is active, the app will navigate to the new flow.

### Testing information
TC1
1. Install & Activate the WCS&T plugin on your store
2. Create an order with processing state that is available to purchase shipping labels
3. Check that tapping on create shipping label navigates to the old flow

TC2
1. Install & Activate the WCS plugin on your store
2. Create an order with processing state that is available to purchase shipping labels
3. Check that tapping on create shipping label navigates to the new flow (on debug build)

### The tests that have been performed
The old plugin navigates to the old experience while the new navigates to the new screen (on debug build)

### Images/gif

https://github.com/user-attachments/assets/645522d5-8fe4-470e-92cb-caf902ebeb94



## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
